docs/request-audit-headers.md

```diff
@@ -25,28 +25,13 @@ For synchronous (i.e. non-bulk) requests, BFD users SHALL include meaningful val
 * `BlueButton-UserId`: the unique ID in the frontend system for the user (e.g. beneficiary) that data is being requested on behalf of.
 * `BlueButton-User`: the human-readable login ID (e.g. email address) in the frontend system for the user (e.g. beneficiary) that the data is being requested on behalf of.
 * `BlueButton-BeneficiaryId`: the unique beneificary ID (e.g. `Patient.id`) from BFD for the user the beneficiary whose data is being requested.
+* `x-forwarded-for`: a standard header for identifying the originating IP address of a client connecting to a web server through an HTTP proxy or a load balancer.
 
 ## Asynchronous Requests
```

Comment:
Do we need another header to differentiate between the various partner applications? Or do we believe that `BULK-CLIENTID` can be leverage to see who the source was (e.g. BCDA request would have BULK-CLIENTID set to A****, V***, E***)

Reply:
We record the TLS certificate presented in the handshake as another header, which we use to key off.